vllm/entrypoints/openai/protocol.py

```diff
@@ -333,7 +333,7 @@ def to_sampling_params(self, default_max_tokens: int) -> SamplingParams:
                 assert json_schema is not None
                 self.guided_json = json_schema.json_schema
                 if self.guided_decoding_backend is None:
-                    self.guided_decoding_backend = "lm-format-enforcer"
+                    self.guided_decoding_backend = "xgrammar"
```

Comment:
The PR title now seems to imply it's just adding a test case, but this line is still included. I assume you left it on purpose, but it'd be nice to call it out.

Sorry to nit-pick, but the PR title is the only thing that makes it into the commit log still ...

Reply:
Thanks, updated the title